homeassistant/components/binary_sensor/zigbee.py

```diff
@@ -0,0 +1,19 @@
+"""
+homeassistant.components.binary_sensor.zigbee
+
+Contains functionality to use a ZigBee device as a binary sensor.
+"""
+from homeassistant.components.zigbee import (
+    ZigBeeDigitalIn, ZigBeeDigitalInConfig)
+
+
+DEPENDENCIES = ["zigbee"]
+
+
+def setup_platform(hass, config, add_entities, discovery_info=None):
+    """
+    Create and add an entity based on the configuration.
+    """
+    add_entities([
+        ZigBeeDigitalIn(hass, ZigBeeDigitalInConfig(config))
```

Comment:
The binary sensor entity should inherit from the binary sensor component. By using the `ZigBeeDigitalIn` entity you are inheriting from `ToggleEntity` which is incorrect.

You should make a base class for ZigBee entities that can handle certain stuff. Then use this via multiple inheritance.

For example for the binary sensor, you should only implement the `is_on` property. The rest like `state` is done by the BinarySensor Entity.